src/main/pages/che-7/end-user-guide/proc_invoking-factories.adoc

```diff
@@ -0,0 +1,58 @@
+// included in:
+// 
+// * assembly_automating-workspace-generation-using-factories
+
+[id="invoking-factories_{context}"]
+= Invoking factories
+
+NOTE: Try a factory by cloning a public workspace on `che.openshift.io`: link:https://che.openshift.io/f?id=factorymtyoro1y0qt8tq2j[try a factory].
```

Comment:
Has anyone tried this factory? it results in `Factory with id 'factorymtyoro1y0qt8tq2j' doesn't exist`
`